### PR TITLE
Add spomorski@ to the tektoncd org

### PR DESCRIPTION
**Pull request by spomorski**

### CODE REVIEW

File: org/org.yaml [hunk 1]
```diff
@@ -67,3 +67,4 @@ orgs:
     - vincent-pli
     - vtereso
     - wlynch
+    -spomorski
```
Comment (sbwsg) [inline org/org.yaml]: minor nit: keep the alphabetical ordering & add a space after the `-`?

Comment (spomorski) [reply org/org.yaml]: Oops, fixed and fixed.

**Pull request merged**